index.bs

```diff
@@ -1927,11 +1941,7 @@ The user agent MUST provide the following new events. Registration for and firin
 
 The user agent MAY fire a <dfn event for="XR">devicechange</dfn> event on the {{XR}} object to indicate that the availability of [=/XR device=]s has been changed. The event MUST be of type {{Event}}.
 
-<section class="unstable">
-A user agent MAY dispatch a <dfn event for="XRSession">blur</dfn> event on an {{XRSession}} to indicate that presentation to the {{XRSession}} by the page has been suspended by the user agent, OS, or XR hardware. While an {{XRSession}} is <dfn>blurred</dfn> it remains active but it may have its frame production throttled. This is to prevent tracking while the user interacts with potentially sensitive UI. For example: The user agent SHOULD blur the presenting application when the user is typing a URL into the browser with a virtual keyboard, otherwise the presenting page may be able to guess the URL the user is entering by tracking their head motions. The event MUST be of type {{XRSessionEvent}}.
-
-A user agent MAY dispatch a <dfn event for="XRSession">focus</dfn> event on an {{XRSession}} to indicate that presentation to the {{XRSession}} by the page has resumed after being suspended. The event MUST be of type {{XRSessionEvent}}.
-</section>
+A user agent MAY dispatch a <dfn event for="XRSession">visibilitychange</dfn> event on an {{XRSession}} to indicate that the [=XRSession/visibility state=] of the {{XRSession}} has changed. The event MUST be of type {{XRSessionEvent}}.
```

Comment:
Should we require it be dispatched whenever the value of `{{XRSession/visibilityState}}` changes?

We probably need this MAY so that UAs aren't required to support noticing changes in visibility state, but if they can report these changes through the attribute they should also trigger the event.

Reply:
You're absolutely right. I've update the PR so that changes to the attribute MUST trigger the event, but that the attribute only SHOULD be affected by platform-level changes, because I don't think we can require that in all cases.

Reply:
Yeah, we shouldn't require UAs to notice everything.